internal/converter/internal/promtailconvert/promtailconvert.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/grafana/agent/internal/component/common/loki"
 	"github.com/grafana/agent/internal/converter/diag"
 	"github.com/grafana/agent/internal/converter/internal/common"
-	"github.com/grafana/agent/internal/converter/internal/promtailconvert/internal/build"
+	"github.com/grafana/agent/internal/converter/internal/promtailconvert/build"
```

Comment:
These files all had to move in order to let the otelcol converter make use of them